RebelAllianceBank/Users/Customer.cs

```diff
@@ -113,46 +113,72 @@ public void CreateAccount()
         /// <summary>
         /// A method to transfer from one of current users account to another users account.
         /// </summary>
-        /// <param name="currentUser">Input personal number of logged in user</param>
         /// <param name="users">A list of all users</param>
         public void TransferUserToUser(List<IUser> users)
         {
+            // Checks if user have any bank accounts at all.
+            if (_bankAccounts == null || !_bankAccounts.Any())
+            {
+                Console.WriteLine("Du har inga bankkonton att göra en överföring från. Tryck på 'enter' för att återgå till menyn.");
+                while (Console.ReadKey(true).Key != ConsoleKey.Enter) { }
+                return;
+            }
             Customer otherUser = null;
             IBankAccount otherAccount = null;
             IBankAccount currentUserAccount = null;
 
             while (currentUserAccount == null)
             {
-                Console.WriteLine("Skriv namnet på det konto du vill föra över från:");
+                Console.WriteLine("Skriv namnet på det konto du vill föra över från, skriv 'avsluta' för att återgå till menyn");
                 foreach (var account in _bankAccounts)
                 {
                     Console.WriteLine($"{account.AccountName} (Saldo: {account.Balance:N2})");
                 }
                 string currentUserAccountName = Console.ReadLine();
                 currentUserAccount = _bankAccounts.FirstOrDefault(account => account.AccountName.Equals(currentUserAccountName, StringComparison.OrdinalIgnoreCase));
-                if (currentUserAccount == null)
+                if ("avsluta" == currentUserAccountName.ToLower())
```

Comment:
You still need to catch null somehow in your if-statements. Right now, null. crashes the program

Reply:
Fixed! now it should work as intended